apps/modernization-ui/src/apps/patient/add/extended/sections.ts

```diff
@@ -0,0 +1,17 @@
+export type Section = {
+    id: string;
+    label: string;
+};
```

Comment:
Issue: Duplicate type for section. Could just use the new `NavSection` defined in `SectionNavigation`